sdk/openai/Azure.AI.OpenAI/tests/OpenAIImageTests.cs

```diff
@@ -50,6 +50,16 @@ public async Task CanGenerateImages(Service serviceTarget)
             ImageGenerationData firstImageLocation = imageGenerations.Data[0];
             Assert.That(firstImageLocation, Is.Not.Null);
             Assert.That(firstImageLocation.Url, Is.Not.Null.Or.Empty);
+
+            // ContentFilterResults and PromptFilterResults are currently Azure Only
+            if (serviceTarget is Service.Azure)
+            {
+                foreach (ImageGenerationData data in imageGenerations.Data)
+                {
+                    Assert.That(data.ContentFilterResults, Is.Null.Or.Empty);
```

Comment:
Shouldn't it be: 

```suggestion
                    Assert.That(data.ContentFilterResults, Is.Not.Null.Or.Empty);
```

Reply:
Resolved by https://github.com/ninghu/azure-sdk-for-net/pull/16/commits/b314ffb7ec0523073271974da608f9c3018e2613